pkg/loadbalancing/dnslb/impl.go

```diff
@@ -0,0 +1,166 @@
+package dnslb
+
+import (
+	crand "crypto/rand"
+	"crypto/tls"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	mrand "math/rand"
```

Comment:
why not use crypto/rand for everything?

Reply:
To avoid draining the entropy pool. Some systems pull directly from the entropy pool rather than a PRNG.